editor/project_manager/project_list.cpp

```diff
@@ -242,6 +256,8 @@ ProjectListItemControl::ProjectListItemControl() {
 	favorite_button->set_mouse_filter(MOUSE_FILTER_PASS);
 	favorite_box->add_child(favorite_button);
 	favorite_button->connect(SceneStringName(pressed), callable_mp(this, &ProjectListItemControl::_favorite_button_pressed));
+	favorite_button->connect(SceneStringName(focus_entered), callable_mp((CanvasItem *)favorite_button, &CanvasItem::set_self_modulate).bind(favorite_focus_color));
```

Comment:
That still won't work. The Color is bound as copy, not reference, so it won't get updated.
You could instead add a method `_button_focus_changed(Button *p_button, bool p_focus)` and bind the button and true/false for focus and fetch the color in the method.

Reply:
Yes, the new method also needs to be called when `NOTIFICATION_THEME_CHANGED`.